extensions/amp-ima-video/0.1/amp-ima-video.js

```diff
@@ -84,6 +87,16 @@ class AmpImaVideo extends AMP.BaseElement {
     user().assert(isExperimentOn(this.win, TAG),
         'Experiment ' + TAG + ' is disabled.');
 
+    this.viewport_ = this.getViewport();
+    if (this.element.getAttribute('data-delay-ad-request') === 'true') {
+      this.viewport_.onScroll(() => { this.sendCommand_('onScroll'); });
```

Comment:
My understanding is on the very first scroll we want to init ads but subsequent scroll events are no-ops. If tha's the case I suggest:

1- renaming to `onFirstScroll`
2- `onScroll` returns a `unlistener`, let's call it after sendCommand so this only fires once.

Reply:
Done